### DISCUSSION TITLE
My account was hacked

### DISCUSSION THREAD

**Discussion by ilzmatias** [community/community#126610] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello! Can someone please respond? I want to unlink my academic email from the GitHub account matiasilz as it was hacked and used for spam. I want to access the GitHub Education benefits since I am in university, but I can't link this GitHub account with my email. I can provide the necessary information to verify my identity. I have activated two-factor authentication on this account, which I did not have enabled on the matiasilz account. Thank you for your attention. Best regards!

**Comment by Wid1ey** [comment]
To address the issue of unlinking your academic email from your hacked GitHub account and accessing GitHub Education benefits, follow these steps:

### Contact GitHub Support

1. **Prepare Your Information**:
   - Your full name.
   - Your current GitHub username (matiasilz).
   - Your academic email address.
   - Any other email addresses associated with your GitHub account.
   - A description of the issue (hacked account used for spam).
   - Mention that you have enabled two-factor authentication on your account.
   - Proof of identity (e.g., a scan of your student ID or a letter from your university).

2. **Submit a Support Request**:
   - Go to the [GitHub Support Contact Form](https://support.github.com/contact).
   - Choose "Account Recovery" as the reason for your contact.
   - Provide all the information prepared in step 1 in the description field.
   - Specify that you want to unlink your academic email from the hacked account and link it to a new or existing GitHub account.

### Example Message to GitHub Support

Here is a template you can use when contacting GitHub support:

```markdown
Subject: Urgent: Unlink Academic Email from Hacked GitHub Account

Dear GitHub Support Team,

I am writing to request assistance with unlinking my academic email from my GitHub account (matiasilz), which was recently hacked and used for spam. I am currently enrolled in university and need to access the GitHub Education benefits. However, I cannot link my academic email to my GitHub account due to the hacking incident.

Here are the details of my account and the situation:

- **Current GitHub Username**: matiasilz
- **Academic Email Address**: [your_academic_email@example.com]
- **Additional Email Addresses**: [other_emails@example.com]
- **Two-Factor Authentication**: Enabled on the matiasilz account

Proof of Identity:
- Attached is a scan of my student ID for verification purposes.
- I can provide any other necessary information to verify my identity and ownership of the account.

I kindly request that my academic email be unlinked from the hacked account and linked to a new or existing GitHub account. This will allow me to continue using GitHub services securely and access the GitHub Education benefits.

Thank you for your attention and assistance.

Best regards,
[Your Full Name]
[Your University]
[Your Contact Information]
```

### Follow Up

- After submitting your request, keep an eye on your email for any response from GitHub support.
- They may ask for additional information or provide instructions for verifying your identity and resolving the issue.

### Preventive Measures

- **Enable Two-Factor Authentication (2FA)**: Ensure that 2FA is enabled on all your accounts to provide an additional layer of security.
- **Use Strong Passwords**: Use strong, unique passwords for each of your online accounts.
- **Monitor Account Activity**: Regularly check your account activity for any unauthorized actions and report them immediately.

By following these steps, you should be able to unlink your academic email from the hacked account, link it to a secure account, and access your GitHub Education benefits.

**Reply by ilzmatias** [reply]
Thank you so much!! I really appreciate you taking the time to respond to me. You helped me a lot!!

**Comment by ebndev** [comment]
Hi @ilzmatias,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
